packages/amplify_api_plugin_interface/lib/src/types/pagination/paginated_result.dart

```diff
@@ -17,21 +17,45 @@ import 'package:amplify_core/amplify_core.dart';
 
 import '../../types.dart';
 
-abstract class PaginatedResult<T extends Model> extends Model {
+class PaginatedResult<T extends Model> extends Model {
+  /// Model instances for this set of results. An entry might be null if
```

Comment:
*nit:* https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

```suggestion
  /// Model instances for this set of results.
  ///
  /// An entry might be null if
```

Reply:
done, thanks